geometry_msgs/Polygon polygon
geometry_msgs/Vector3 vector
---
mesh_msgs/PolygonMesh mesh
